changes/watch_linux.go

```diff
@@ -0,0 +1,53 @@
+package changes
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"golang.org/x/exp/inotify"
```

Comment:
Just FYI, fsnotify is based on x/exp/inotify, but more than likely has fixed a few bugs over the years. I have been thinking of splitting out a github.com/fsnotify/inotify or github.com/fsnotify/fsnotify/inotify package at some point. I'll be sure to let you know if that ever gets done. :-)